backend/models/user.py

```diff
@@ -0,0 +1,45 @@
+class User:
+    def __init__(self, FName, LName, Email, Location, Role, PetId, Avatar, Id=""):
```

Comment:
PetId is optional, so it should be PetId = []
I think Avatar is optional as well